repos/fdbt-site/src/pages/selectCapValidity.tsx

```diff
@@ -8,18 +8,30 @@ import { getSessionAttribute } from '../utils/sessions';
 import { getAndValidateNoc, getCsrfToken, getErrorsByIds } from '../utils';
 import RadioConditionalInput from '../components/RadioConditionalInput';
 import { isCapExpiry } from '../interfaces/typeGuards';
-import { getFareDayEnd } from '../data/auroradb';
+import { getCapExpiry, getFareDayEnd } from '../data/auroradb';
+import BackButton from '../components/BackButton';
 
 const title = 'Cap Validity - Create Fares Data Service';
 const description = 'Cap Validity selection page of the Create Fares Data Service';
 
+export const validityHintText: { [key: string]: string } = {
+    endOfCalendarDay: 'The cap applies to journeys made before midnight',
+    '24hr': 'The cap applies to journeys made within 24hrs of the first tap',
+    fareDayEnd: "The cap applies to journeys made during the 'fare day' as defined by your business rules",
+};
+
 interface CapValidityProps {
+    capExpirySelected: string;
     errors: ErrorInfo[];
     fieldset: RadioConditionalInputFieldset;
     csrfToken: string;
 }
 
```

Comment:
If I select fare day end, and then edit it, the page doesn't show fare day end by default.

Reply:
The above isn't happening anymore but if I change the expiry from fare day end to something else, then edit it, it still shows fare day end as the selected value.

Reply:
tested the scenario and working fine now...